src/HomePage/Heading/Heading.js

```diff
@@ -3,8 +3,11 @@ import "./Heading.scss";
 
 const Heading = () => {
   return (
-    <div>
-      <h1 className="heading">Hello World, I'm Anita</h1>
+    <div className="header">
```

Comment:
Im not sure header is the right name for this, this is more like main section or heading container or something. Header to me suggests something right at the top of the page like a navabar

Reply:
Have amended